linux-tkg-config/prepare

```diff
@@ -640,6 +649,14 @@ CONFIG_DEBUG_INFO_BTF_MODULES=y\n
       sed -i -e 's/CONFIG_GCC_PLUGINS=y/# CONFIG_GCC_PLUGINS is not set/' ./.config
     fi
   fi
+  # Clang LTO
+  if [ "$_compiler_name" = "-llvm" ] && [ $_basever -ge 512 ]; then
+    if [ "$_lto_mode" = "full" ]; then
+      echo "CONFIG_LTO_CLANG_FULL=y" >> ./.config
```

Comment:
I think from now on it's better to use:
```
scripts/config -e LTO_CLANG_FULL
scripts/config -d LTO_CLANG_THIN
```
Instead of `echo` or `sed`. Since these options can already be in the config file (and that will happen for `install.sh` installs in the future), `scripts/config` knows when to add or replace existing lines in `.config`.